lib/executor/common_test.go

```diff
@@ -0,0 +1,56 @@
+package executor
```

Comment:
All `*_test.go` files should also have the license header, right?

Reply:
@imiric please add.

Reply:
Resolved in 6dc8675e.